cockroachcloud/authorization.md

```diff
@@ -23,20 +23,20 @@ See:
 Users may be granted two levels of access:
 
 - 'Developer' allows users to:
-  - Read cluster information in the CockroachDB Cloud Console and Cloud API
+  - Read cluster information using the CockroachDB Cloud Console and Cloud API
 - 'Administrator' allows users to:
   - Create and delete organization users
-  - Create, modify and delete clusters
+  - Create, modify, and delete clusters
   - Create and manage SQL roles on particular clusters
 
-Within a CockroachDB Cloud Organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed over the cluster.
+Within an organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed across the cluster.
 
-Each cluster also has its own set of SQL roles, which allow authenticated users to execute SQL statements via any PostgreSQL-compatible client.
+Each cluster has its own set of SQL roles, which allow authenticated users to execute SQL statements using any PostgreSQL-compatible client.
 
 ## Cluster roles for organization users using Cluster SSO
 
 Cluster Single Sign-On (SSO) for {{ site.data.products.db }} allows authorized organization users to directly access clusters within the organization via [`ccloud`](ccloud-get-started.html), the {{ site.data.products.db }} command line interface.
 
 However, because organization roles and roles on any given SQL cluster are logically separate, a corresponding SQL role must be created for each SSO organization user, on each particular cluster.
 
-This correspondence lies in the SQL role name, which must be: `sso_{email_name}`, where 'email_name' refers to everything up to the '@'' in an email address. For example, the SQL user `sso_docs` would result from docs@cockroachlabs.com. `ccloud will prompt you to create this user if it does not already exist, in which case an admin must create it manually.
\ No newline at end of file
+This correspondence lies in the SQL role name, which must be: `sso_{email_name}`, where 'email_name' refers to everything up to the '@'' in an email address. For example, the SQL user `sso_docs` would result from docs@cockroachlabs.com. `ccloud` prompts you to create this user if it does not already exist, in which case an admin must create it manually.
```

Comment:
@nickvigilante Is it a bug to look within code font for spelling errors?

Reply:
I believe it is. Can you log it and assign it to me?

Reply:
https://cockroachlabs.atlassian.net/browse/DOC-7324